tests/testsuite/standard_lib.rs

```diff
@@ -323,6 +329,33 @@ fn check_core() {
         .run();
 }
 
+#[cargo_test(build_std_mock)]
+fn test_std_on_unsupported_target() {
+    let setup = setup();
+
+    let p = project()
+        .file(
+            "src/main.rs",
+            r#"
+        fn main() {
+            println!("hello");
+        }
+        "#,
+        )
+        .build();
+
+    p.cargo("build")
+        .build_std(&setup)
+        .target("aarch64-unknown-none")
+        .with_status(101)
+        .with_stderr_data(
+            "\
+[ERROR] building std is not supported on this target: [..]
+",
```

Comment:
Unless intentionally, would recommend wrapping with `str![[...]]` so that the snapshot could get auto-updated.

Reply:
Noted, will make this change.